src/EditorFeatures/Test/Options/GlobalOptionsTests.cs

```diff
@@ -99,6 +95,12 @@ public void SetGlobalOption(OptionKey2 optionKey, object? value)
         public bool SetGlobalOptions(ImmutableArray<KeyValuePair<OptionKey2, object?>> options)
             => throw new NotImplementedException();
 
+        public void AddOptionChangedHandler(object target, EventHandler<OptionChangedEventArgs> handler)
+            => throw new NotImplementedException();
+
+        public void RemoveOptionChangedHandler(object target, EventHandler<OptionChangedEventArgs> handler)
+            => throw new NotImplementedException();
```

Comment:
would it be possible to make two PRs?  one for the refcounted stuff, and one for the weak event handlerS?

Reply:
➡️ Extracted #68062